scripts/create-symlinks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage in the project root folder:
+# ./scripts/create-symlinks.sh PACKAGE_NAME
+#
+# Usage in package folder
+# ../../scripts/create-symlinks.sh
+
+packageDir=".";
+if [ -n "$1" ]
+	then
+		# Override $pageDir if argument was provided.
+		packageDir="./packages/$1";
+
+		# Just in case package folder doesn't exist yet.
+		mkdir -p $packageDir;
```

Comment:
I think this script would make more sense as an `init` script: whenever we want to add a new package we run it, and it creates our package with defaults.

Reply:
`npm run init -- packageName` would this usage make sense? @willclarktech